tools/validators/instance_validator/README.md

```diff
@@ -102,7 +102,8 @@ Run `python instance_validator.py` and provide the following arguments:
 
   * `-t/--timeout` **[Optional]** The timeout duration in seconds for the telemetry validation test. The default value is 600 seconds, or 10 minutes. If this time limit is exceeded before the validator receives a test pubsub message for each of the entities configured in the given instance config file, the test will fail with an error and report the entities that were not heard from.
 
-  * `--udmi` **[Optional]** Treat message stream on PubSub subscription as [UDMI](https://github.com/faucetsdn/udmi/). **NOTE:** This is required for telemetry validation when devices implement the UDMI specification. Set to true by default.
+  * `--udmi` **[Optional]** Validates entity metadata as [UDMI](https://github.com/faucetsdn/udmi/). Include argument when `True` and do not include when `False`.
```

Comment:
keep the default for True?

Reply:
This was confusing for Brian. I think it's more straight forward if the flag is included, then the argument is true. Otherwise if the flag is not included the argument is false. 

Previously, the `default` and `action` arguments for `parser.add_argment` were conflicting and the `--udmi` flag was always true.

Reply:
since the msi are mainly using it for udmi, I prefer to keep the udmi mode set to true by default. wdyt?

Reply:
Done.